package.json

```diff
@@ -392,6 +475,99 @@
+            "description": "Enable syntax highlighting and status bar",
+            "scope": "resource"
+          },
+          "vscode-dfdl.showPath": {
+            "type": "boolean",
+            "default": true,
+            "description": "Show tag path in status bar",
+            "scope": "resource"
+          },
+          "vscode-dfdl.showRuler": {
+            "type": "boolean",
+            "default": true,
+            "description": "Show highlighted tag pair in ruler section",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightSelfClosing": {
+            "type": "boolean",
+            "default": false,
+            "description": "Highlight self-closing tags",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightFromContent": {
+            "type": "boolean",
+            "default": true,
+            "description": "Highlight matching tag from inside tag content",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightFromName": {
+            "type": "boolean",
+            "default": true,
+            "description": "Highlight matching tag from the tag name",
+            "scope": "resource"
+          },
+          "vscode-dfdl.highlightFromAttributes": {
+            "type": "boolean",
+            "default": true,
+            "description": "Highlight matching tag from the tag attributes",
+            "scope": "resource"
+          },
+          "vscode-dfdl.noDefaultEmptyElements": {
+            "type": "boolean",
+            "default": false,
+            "description": "Don't use default HTML empty elements",
+            "scope": "resource"
+          },
+          "vscode-dfdl.customEmptyElements": {
+            "type": "array",
+            "default": null,
+            "description": "Custom empty elements in addition to the default HTML empty elements",
+            "scope": "resource"
+          },
+          "vscode-dfdl.styles": {
+            "opening": {
+              "left": {
+                "underline": "yellow"
+              },
+              "right": {
+                "surround": "#155FFA"
+              },
+              "name": {
+                "highlight": "rgba(180, 20, 80, 0.3)"
+              }
+            },
+            "closing": {
+              "full": {
+                "custom": {
+                  "dark": {
+                    "borderWidth": "0 0 1px 0",
+                    "borderColor": "white",
+                    "borderStyle": "solid",
+                    "borderRadius": "4px",
+                    "right": "10px"
+                  },
+                  "light": {
+                    "borderWidth": "0 0 1px 0",
+                    "borderColor": "brown",
+                    "borderStyle": "solid",
+                    "borderRadius": "4px",
+                    "right": "10px"
+                  }
+                }
+              }
+            }
+          }
+        }
```

Comment:
The above comment is correct that be removed, but the hightlighting style I believe should stay as it these will use our custom set colors instead of the defaults

Reply:
I found that this is just copied straight from the second example in the [Style Examples](https://github.com/vincaslt/vscode-highlight-matching-tag/blob/b15dc1f0836d74fe110f6275e59374d2eb870cbb/README.md#styling-examples=) section of the hightlight matching tags section.

Seems this is something a user should edit if they want different defaults than what that extension defines, and not something our extension should prescribe.

There are also potentially licensing implications, and if we are going to copy this we need to mention it in the LICENSE/NOTICE. Though, one could maybe argue this is a trival change that isn't copyrightable. Regardless, I don't think we should need it.